### PR TITLE
Add CSRF token, change ImageProxy type from user to system, and add CustomJS extension

### PR DESCRIPTION
**Pull request by Frenzie**
This is for https://github.com/FreshRSS/FreshRSS/issues/1253

CustomJS is CustomCSS' natural counterpart.

### CODE REVIEW

File: xExtension-CustomJS/metadata.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"name": "Custom JS",
+	"author": "Marien Fressinaud",
```
Comment (Alkarex) [inline xExtension-CustomJS/metadata.json]: You should have your own name there @Frenzie :-)

**Comment by Alkarex** [comment]
Looks very good, thanks :+1: 
Did you check that it works with the additional CSRF token?

File: xExtension-CustomJS/metadata.json (hunk 1 above)
Comment (Frenzie) [inline xExtension-CustomJS/metadata.json]: I didn't really write it but if you say so. =P

**Comment by Frenzie** [comment]
Yeah, CustomCSS, CustomJS and ImageProxy are all working fine. =)

**Pull request merged**